code/modules/projectiles/projectile/magic_projectiles.dm

```diff
@@ -324,8 +324,11 @@
 				S.icon = change.icon
 				if(H.mind)
 					H.mind.transfer_to(S)
-					to_chat(S, "<span class='warning'>You are an animated statue. You cannot move when monitored, but are nearly invincible and deadly when unobserved!</span>")
-					to_chat(S, "<span class='userdanger'>Do not harm [firer.name], your creator.</span>")
+					var/list/messages = list()
+					messages.Add("<center><span class='userdanger'>You are now an animated statue.</span></center>")
+					messages.Add("<center>You cannot move when monitored, but are nearly invincible and deadly when unobserved! Hunt down those who shackle you.</center>")
```

Comment:
Text is already center aligned in chat_boxes 
```
.boxed_message {
  background: #f7fcff;
  border: 1px solid #111a26;
  margin: 0.5em;
  padding: 0.5em 0.75em;
  text-align: center;
}
```
```suggestion
					messages.Add("You cannot move when monitored, but are nearly invincible and deadly when unobserved! Hunt down those who shackle you.")
```

Reply:
Didnt know that! I had the center on there before I decided to put it in the box (or knew how).